py3status/modules/mpd_status.py

```diff
@@ -110,6 +111,28 @@ class Py3status:
     state_play = '[play]'
     state_stop = '[stop]'
 
+    def _mpdc(self):
+        try:
+            if self.c is None:
+                self.c = MPDClient()
+                self.c.connect(host=self.host, port=self.port)
+                if self.password:
```

Comment:
I think we'd like `self.connection` instead because we'll see `self.c` everywhere.